core/src/main/java/hudson/security/FullControlOnceLoggedInAuthorizationStrategy.java

```diff
@@ -36,30 +37,53 @@
 
 /**
  * {@link AuthorizationStrategy} that grants full-control to authenticated user
- * (other than anonymous users.)
+ * and optionally read access to anonymous users
  *
  * @author Kohsuke Kawaguchi
  */
 public class FullControlOnceLoggedInAuthorizationStrategy extends AuthorizationStrategy {
+    /**
+     * Whether to allow read access, default behavior
+     * previously was true
+     */
+    private boolean allowAnonymousRead = true;
```

Comment:
:bug: this will be deserialized as `false` for existing instances loaded from disk.

Your options are to invert the sense of the option; make it a `Boolean`, treating `null` like `TRUE` in the getter; or make it a `Boolean` and add a `readResolve` to convert `null` to an explicit `TRUE`.